### ISSUE TITLE
Native Display Implementation Ionic 4

### ISSUE THREAD

**Issue by garauz23** [CleverTap/clevertap-cordova#75]
Hi guys, I'm trying to implement Native Display on Ionic 4 Project, I added an event listener on app.component.ts, then add into a custom banner component the call of getAllDisplayUnits, everything works fine if the user applies for native display campaign however if the user doesn't apply app just crash with the following error "java.lang.NullPointerException: Attempt to invoke virtual method 'int java.util.ArrayList.size()' on a null object reference".

Also event listener never is fired.

I would like to know if I'm doing something wrong with Native Display implementation.

```
   // app.component.ts
    document.addEventListener('onCleverTapDisplayUnitsLoaded', (event: any) => {
      console.log('onCleverTapDisplayUnitsLoaded');
      console.log(event.units);
      console.log(event);
    }, false);
```

```
// my  custom banner component
this.clevertap.getAllDisplayUnits()
    .then((units) => {
      console.log('displayUnits', units);
    }).catch((e) => {
      console.log('error', e);
    });
```

```
2020-11-02 17:37:43.680 28673-28989/com.xxxx.xxx.xxxx E/AndroidRuntime: FATAL EXCEPTION: pool-22-thread-3
    Process: com.xxx.xxxx.xxxxxxxx, PID: 28673
    java.lang.NullPointerException: Attempt to invoke virtual method 'int java.util.ArrayList.size()' on a null object reference
        at com.clevertap.cordova.CleverTapPlugin.displayUnitListToJSONArray(CleverTapPlugin.java:2683)
        at com.clevertap.cordova.CleverTapPlugin.access$900(CleverTapPlugin.java:57)
        at com.clevertap.cordova.CleverTapPlugin$88.run(CleverTapPlugin.java:1950)
        at java.util.concurrent.ThreadPoolExecutor.processTask(ThreadPoolExecutor.java:1187)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:929)
```

**Comment by darshanclevertap** [comment]
@garauz23 Thank you for bringing this to our notice, please allow me some time to replicate this on my end and I will get back to you on the steps ahead. Also closing #74 as two issues have been created.

**Comment by darshanclevertap** [comment]
@garauz23 Sorry for the delay. We will be doing a bug fix release for this in a couple of days. Please stay tuned for more details.

**Comment by garauz23** [comment]
Thanks for your answer, I stay tuned

**Comment by darshanclevertap** [comment]
@garauz23 This has been fixed with v2.3.1. Thank you for your patience.
